### PR TITLE
[CSL-1720] Don't call BListener during lrc

### PR DESCRIPTION
**Pull request by pva701**

### CODE REVIEW

File: wallet/src/Pos/Wallet/Web/Tracking/Sync.hs [hunk 1]
```diff
@@ -503,19 +503,19 @@ getEncInfo encSK = do
...
+selectOwnAddresses encInfo getAddr =
+    mapMaybe (\a -> (a,) <$> decryptAddress encInfo (getAddr a))
 
 setLogger :: HasLoggerName m => m a -> m a
 setLogger = modifyLoggerName (<> "wallet" <> "sync")
 
-decryptAccount :: (HDPassphrase, CId Wal) -> Address -> Maybe CWAddressMeta
-decryptAccount (hdPass, wCId) addr = do
+decryptAddress :: (HDPassphrase, CId Wal) -> Address -> Maybe CWAddressMeta
+decryptAddress (hdPass, wCId) addr = do
```
Comment (gromakovsky) [inline wallet/src/Pos/Wallet/Web/Tracking/Sync.hs]: I don't get why these changes are in this commit.

Comment (pva701) [reply wallet/src/Pos/Wallet/Web/Tracking/Sync.hs]: hz, I noticed and decided to fix it

**Comment by pva701** [comment]
@gromakovsky fixed tests
@neongreen review plz (at least this commit https://github.com/input-output-hk/cardano-sl/pull/1686/commits/ce1e05ccaa4bed17588033de4b1a0cd17e9b678b)

File: lib/src/Pos/Util/Config.hs [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 module Pos.Util.Config
        ( embedYamlConfigCT
+       , embedYamlConfig
```
Comment (gromakovsky) [inline lib/src/Pos/Util/Config.hs]: What's the difference between `embedYamlConfigCT` and `embedYamlConfig`? Do we need both?

File: lib/src/Pos/Block/Logic/Internal.hs [hunk 3]
```diff
@@ -153,8 +153,11 @@ normalizeMempool = do
 -- Invariant: all blocks have the same epoch.
 applyBlocksUnsafe
     :: forall ssc ctx m . MonadBlockApply ssc ctx m
-    => OldestFirst NE (Blund ssc) -> Maybe PollModifier -> m ()
-applyBlocksUnsafe blunds pModifier = do
+    => ShouldCallBListener
+    -> OldestFirst NE (Blund ssc)
+    -> Maybe PollModifier
+    -> m ()
+applyBlocksUnsafe bsc blunds pModifier = do
```
Comment (gromakovsky) [inline lib/src/Pos/Block/Logic/Internal.hs]: bsc? Maybe `scb`?

File: lib/src/Pos/Util/Config.hs (hunk 2 above)
Comment (pva701) [reply lib/src/Pos/Util/Config.hs]: I left `embededYamlConfigCT` just in case, it may be needed in future

**Comment by pva701** [comment]
@gromakovsky approve?

**Comment by pva701** [comment]
approve najmi

File: auxx/src/Command/Rollback.hs [hunk 4]
```diff
@@ -57,7 +58,7 @@ rollbackAndDump numToRollback outFile = do
             liftIO $ BSL.writeFile outFile (serialize allTxs)
             logInfo $ sformat ("Dumped "%int%" transactions to "%string)
                       (length allTxs) (outFile)
-            rollbackBlocksUnsafe (BypassSecurityCheck True) blunds
+            rollbackBlocksUnsafe (BypassSecurityCheck True) (ShouldCallBListener False) blunds
```
Comment (gromakovsky) [inline auxx/src/Command/Rollback.hs]: ???

Comment (pva701) [reply auxx/src/Command/Rollback.hs]: chto ti hochech, zachem vizivat ego tut?

Comment (gromakovsky) [reply auxx/src/Command/Rollback.hs]: What's the reason not to call it?

Comment (gromakovsky) [reply auxx/src/Command/Rollback.hs]: Imo, the only reason not to call BListener is when we are sure we will apply the same blocks back. But here it's not the case.

Comment (pva701) [reply auxx/src/Command/Rollback.hs]: ok, fixed. no hz zachem blistener should know about rollback in auxx
